econ/terminate_test.go

```diff
@@ -543,13 +543,13 @@ var agentTests = []struct {
 				Interest:  bigFromStr("0"),
 			},
 		},
-		maxBorrowAndSeal:     bigFromStr("361258756990414938951"),
-		maxBorrowAndWithdraw: bigFromStr("90314689247603734738"),
+		maxBorrowAndSeal:     bigFromStr("355241600312332747173"),
+		maxBorrowAndWithdraw: bigFromStr("88308970354909670812"),
 		liquidationValue:     bigFromStr("120419585663471646317"),
-		borrowLimit:          bigFromStr("359258756990414938951"),
-		withdrawLimit:        bigFromStr("117752918996804979651"),
-		marginCall:           bigFromStr("2352941176470588235"),
-		leverageRatio:        0.0166085939341235,
+		borrowLimit:          bigFromStr("353235881419638683247"),
+		withdrawLimit:        bigFromStr("57665444302030819344"),
+		marginCall:           bigFromStr("2359669285522428148"),
+		leverageRatio:        0.016656085317377764,
```

Comment:
@ganzai-san a couple test values changed - i think this is generally to be expected